script.sublissimo/changelog.txt

```diff
@@ -0,0 +1,2 @@
+v0.0.1
```

Comment:
No need to submit this file. please use the `<news>` tag in addon.xml: https://kodi.wiki/view/Add-on_structure#changelog.txt